cla_backend/apps/knowledgebase/tests/test_csv_2_fixture.py

```diff
@@ -28,1416 +47,724 @@ def slice_records(self, output_records, expected_records, start_index=None, end_
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 12,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Employment",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 13,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Immigration and asylum",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 14,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Mental health",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 15,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Miscellaneous",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 16,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Personal injury",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 17,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Public",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-            {
-                u"pk": 18,
-                u"model": u"knowledgebase.articlecategory",
-                u"fields": {
-                    u"name": u"Generic",
-                    u"created": u"{date}".format(date=self.datetime_now),
-                    u"modified": u"{date}".format(date=self.datetime_now),
-                },
-            },
-        ]
+        csv_file = self.find_path_to_csvfile("empty_csv")
+        file = open(csv_file)
+        try:
+            csv = KnowledgebaseCsvParse(file)
+        except SystemExit:
+            self.fail("SystemExit Exception with a error code of -1")
+        else:
```

Comment:
I would probably keep this test but just assert how many objects are described (rather than checking their actual data), so just check there are 18 categories and no articles or matrices